From 624bb7cadb895f1e6377a6c33953e93239bdb1eb Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Tue, 12 Dec 2017 05:49:11 -0600
Subject: [PATCH] "0001-libhwui-fixes-and-symbols-for-libhwuibp
 _______________________________

*Previous From 258d77921d91b6786706d7ec326c6d9146b4d902 Mon Sep 17 00:00:00 2001
*Previous From: Meticulus <theonejohnnyd@gmail.com>
*Previous Date: Tue, 14 Nov 2017 04:59:06 -0600
*Previous Subject: [PATCH] libhwui: fixes and symbols for libhwuibp
*
*-define out unused code and add missing symbol referenced
*by libhwuibp
*
*Change-Id: I5237e41f7155349967d7fc22f769dbce2d9b150e"
---
 libs/hwui/ProgramCache.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/libs/hwui/ProgramCache.cpp b/libs/hwui/ProgramCache.cpp
index 8cc0aa7..526ef52 100644
--- a/libs/hwui/ProgramCache.cpp
+++ b/libs/hwui/ProgramCache.cpp
@@ -905,3 +905,7 @@ void ProgramCache::printLongString(const String8& shader) const {
 
 }; // namespace uirenderer
 }; // namespace android
+
+extern "C" void _ZN7android10uirenderer12ProgramCache14generateShaderERKNS0_18ProgramDescriptionERNS_7String8ES6_(const android::uirenderer::ProgramDescription& description, android::String8& shader) {
+    PROGRAM_LOGD("Meticulus: '%s'\n", shader.string());
+}
-- 
2.7.4

